edge/pkg/edged/edged_status.go

```diff
@@ -332,7 +351,15 @@ func (e *edged) setCPUInfo(total, allocated v1.ResourceList) error {
 }
 
 func (e *edged) getDevicePluginResourceCapacity() (v1.ResourceList, []string) {
-	return e.runtime.GetDevicePluginResourceCapacity()
+	switch e.containerRuntimeName {
+	case DockerContainerRuntime:
+		return e.runtime.GetDevicePluginResourceCapacity()
+	case RemoteContainerRuntime:
+		//resourceList, _, str := e.containerManager.GetDevicePluginResourceCapacity()
```

Comment:
see earlier comment for device plugin support via cri. we can track via an issue. please create the same. thanks.

Reply:
Ok